.github/workflows/graph.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy Graph
+
+on:
+  push:
+    branches: master
+
+jobs:
+  deploy-goerli:
+    runs-on: ubuntu-latest
+    environment: graph
+    steps:
+      - uses: actions/checkout@v3
+
```

Comment:
depending on what we do we'd need to recur submodules here

Reply:
I think would be necessary if we needed to run `forge` on contracts package... here we worked on subgraph folder only ✌🏻